README.md

```diff
@@ -120,9 +120,9 @@ If an external request triggers a process that requires background tasks to run,
 
 ## Current status and roadmap
 
-- **UPCOMING FEATURES:**: In 0.3.0 we introduced the ability for hopeit.engine to use different Stream Manager implementations by configuration. We are planning to create plugins to provide a hopeit.engine base Stream Processing engine to replace Redis Streams to simplify operations in some Scenarios. This can also support Kafka integration in the future.
+- **UPCOMING FEATURES:**: In hopeit.engine 0.3.0 we introduced the ability for hopeit.engine to use different Stream Manager in order to simplify implementations by configuration. We are planning to create plugins to provide a hopeit.engine base Stream Processing engine to replace Redis Streams to simplify operations in some scenarios. Thanks to this, Kafka integration can be supported in the future.
```

Comment:
- **UPCOMING FEATURES:**: `hopeit.engine` 0.3.0 introduced the ability to use different Stream Managers in order and enable custom integrations. We are planning to create plugins to provide a `hopeit.engine` based Stream Processing engine to optionally use instead Redis Streams, simplifying operations in some scenarios. Thanks to this modularization, also Kafka integration can be supported in the future.